dynamodb/src/test/resources/application.conf

```diff
@@ -3,3 +3,5 @@ akka {
   logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
   loglevel = "DEBUG"
 }
+
+akka.http.client.parsing.max-content-length = 20m
```

Comment:
Should it be recommended in the docs to increase this from the Akka HTTP default (8 MByte)?

Reply:
That is only needed for tests. The RetryFlow test sends a request larger than DynamoDb limit of 16mb. And then it partially fails. RetryFlow is then used to retry the failed part.